src/reducers/userReducer.js

```diff
@@ -0,0 +1,21 @@
+const user = (state={}, action) => {
+  switch (action.type) {
+    case 'CREATE_USER':
+      return {
+        state.id: action.id
+      }
+    case 'SIGN_IN_USER':
+      return {
+        state.name: action.name, 
+        state.email: action.email
+      }
+    case 'SIGN_OUT_USER': 
+      return {
+        state.id: ''
+      }
+    default: 
+      return state
+  }
+}
+
+export default user
```

Comment:
Great work on this Rajaa! After reviewing over how we would want to the store our user object, lets also add an id, into the payload for when we sign a user in.